code/modules/events/portal_storm.dm

```diff
@@ -0,0 +1,133 @@
+/datum/round_event_control/portal_storm_syndicate
+	name = "Portal Storm: Syndicate Shocktroops"
+	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
+	weight = 5
```

Comment:
reeeally not sure if this is adequate, i feel like these will just murder everyone roudnstart